### PR TITLE
[ng] upgrade to 3.0.0-beta+1

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #507. Supersedes #538.

### CODE REVIEW

**Comment by chalin** [comment]
@kevmoo - this is ready for review (Travis is now green over all tasks, included the allowed failures).

File: gulp/test.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = function (gulp, plugins, config) {
 
   const runAngularTest = 'pub run angular_test';
 
-  gulp.task('test', ['_test-ng-doc', '_test-ng-test']);
+  gulp.task('test', ['_test-ng-doc']); // Disable for now: , '_test-ng-test'
```
Comment (kevmoo) [inline gulp/test.js]: Any reason why?

Comment (chalin) [inline gulp/test.js]: There are issues with specific pinned versions of bootstrap etc. in the ng-test example. I figured that I'd investigate that later (possibly asking Matan for some input).

FYI, #507 has list item reminder to re-enable this test.

**Comment by chalin** [comment]
@kevmoo - if this is good to go, will you be merging or shall I do it?

File: gulp/test.js (hunk 1 above)
Comment (kevmoo) [inline gulp/test.js]: Please put an reference to the issue inline here – so we keep track of it.

Often we'll do

```
// TODO(patrice) re-enable when #XYZ is fixed
```

or similar

**Comment by kevmoo** [comment]
> @kevmoo - if this is good to go, will you be merging or shall I do it?

please add the TODO note, then you're good to merge

**Comment by chalin** [comment]
TODO note added. Merging.

**Pull request merged**